contrib/sly-package-fu.el

```diff
@@ -155,20 +169,21 @@ places the cursor at the start of the DEFPACKAGE form."
     (save-excursion
       (cl-loop for sexp = (read-sexp) while sexp collect sexp))))
 
+(defun sly-package-normalize-name (name)
```

Comment:
It's a good idea to reuse this code in a function, but please rename it `sly-package-fu--normalize-name` to comply with emacs lisp coding practices.